feature/converter/converter.py

```diff
@@ -0,0 +1,61 @@
+import sys
+
+
+class Converter(object):
+
+    def __init__(self, first_file_name, second_file_name):
+        self.first_file_name = first_file_name
+        self.second_file_name = second_file_name
+        self.first_file = None
+        self.second_file = None
+        self.content = ''
```

Comment:
`self.content = ""
self.first_file = None
self.second_file = None`

Please explain why we create here these variables if they get their data only in **open_file** method :)